src/Cody.VisualStudio/Completions/CodyProposalSourceProvider.cs

```diff
@@ -93,12 +90,6 @@ private void OnSuggestionAccepted(object sender, SuggestionAcceptedEventArgs e)
 
         public async override Task<ProposalSourceBase> GetProposalSourceAsync(ITextView view, CancellationToken cancel)
         {
-            //var list = Input.ToArray();
```

Comment:
Is not used for debugging `//[ImportMany] public IEnumerable<ProposalSourceProviderBase> Input { get; set; }`

I would say it could stay then, as useful debugging help.